docker/test/smoke_test_docker_image.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -e
+set -ae
```

Comment:
@pashagolub This "auto-export" flag is a pretty evil one, maybe better add a comment - I've spent some hours due to these in the past :D

Reply:
Yeah, I can just add `export` to needed vars. What exactly catch was it? :)